tests/test-invalid-weirdcasing-spacecrlf-nonlast.eml

```diff
@@ -0,0 +1,17 @@
+From: John Doe <jdoe@machine.example>
+To: Mary Smith <mary@example.net>
+Subject: Test invalid weirdcasing spacecrlf nonlast
+mssSAGe-iD: < test invalid
```

Comment:
Did you intend for "message" to be spelled wrong on this one?

Reply:
Nope, all them should say "message-id". Good catch. Going to commit the change.

Reply:
Committed at 87deeeb. Closing.